code/workspaces/web-app/src/containers/adminListUsers/AdminUsersContainer.js

```diff
@@ -131,4 +146,24 @@ function AdminUsersContainer() {
   );
 }
 
+const FilterSkeleton = ({ rows = 3, columns = 3 }) => {
```

Comment:
Might it be worth making this a reusable component, e.g. GridFilterSkeleton?

Reply:
Good idea; I have made and used such a component